Microsoft.Toolkit/Helpers/NotifyTaskCompletion.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Helpers
+{
+    /// <summary>
+    /// Helper class to wrap around a Task to provide more information usable for UI databinding scenarios. As discussed in MSDN Magazine: https://msdn.microsoft.com/magazine/dn605875.
+    /// </summary>
+    /// <typeparam name="TResult">Type of result returned by task.</typeparam>
+    public sealed class NotifyTaskCompletion<TResult> : INotifyPropertyChanged
```

Comment:
It's a common MVVM helper.  I'll be using it in the Graph repo for this [PR here](https://github.com/windows-toolkit/Graph-Controls/pull/18/files#diff-fcd960a5aaaaa88ddd1e225f7cfd21b5R22).

Reply:
So shouldn't it be part of that PR instead? I don't see a point to this at all. This type of logic should be handled within your viewmodel already. Exposing the Task to the view doesn't make sense

Reply:
It can't be in that PR as that PR is in a different repository.  See the [blog post]( https://msdn.microsoft.com/magazine/dn605875) about this helper as it explains the benefits of this pattern.  Thanks!